frontend/rust-lib/flowy-database2/src/services/cell/cell_operation.rs

```diff
@@ -128,7 +128,7 @@ pub fn try_decode_cell_to_cell_protobuf(
   field: &Field,
   cell_data_cache: Option<CellCache>,
 ) -> FlowyResult<CellProtobufBlob> {
-  match TypeOptionCellExt::new_with_cell_data_cache(field, cell_data_cache)
```

Comment:
How about TypeOptionCellExt::new() only accept field. and add new method with_cache(cell_data_cache)